src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -40,6 +41,17 @@
         "linux-x64": "Microsoft.Android.Sdk.Linux"
       }
     },
+    "Microsoft.Android.Sdk.NET6": {
+      "kind": "framework",
+      "version": "@NET6_VERSION@",
+      "alias-to": {
+        "osx-x64": "Microsoft.Android.Sdk.Darwin",
+        "osx-arm64": "Microsoft.Android.Sdk.Darwin",
+        "win-x86": "Microsoft.Android.Sdk.Windows",
+        "win-x64": "Microsoft.Android.Sdk.Windows",
+        "linux-x64": "Microsoft.Android.Sdk.Linux"
+      }
+    },
```

Comment:
After looking through some of the workload docs/examples:
* https://github.com/dsplaisted/WorkloadVersioningSample/blob/main/workloads/sdk-manifests/6.0.200/DoomSlug.MonoToolChain.net6/WorkloadManifest.json#L22-L28
* https://github.com/dotnet/sdk/blob/main/documentation/general/workloads/cross-version-workloads.md

I reworked this to be simpler. The `Microsoft.Android.Sdk.$(HostOS)` pack is no longer renamed.

And we simply use these alias mappings instead.